static/src/javascripts/projects/common/modules/onward/related.js

```diff
@@ -66,6 +66,15 @@ define([
                 showCount: false
             }).init();
         } else if (fetchRelated) {
+
+
+            if (ab.isInVariant('RelatedContentDisplayAsRecommendation', 'people-who-read-this-also-read-title')) {
+                var relatedContentSection = document.getElementById('related-content');
+                if (relatedContentSection) {
+                    relatedContentSection.querySelector('.fc-container__header__title').textContent = 'people who have read this also read';
```

Comment:
it would be more standard to use `$` here, but not a blocker. `.textContent` needs to be inside a `fastdom.write` callback though